scss/partials/_base.scss

```diff
@@ -17,9 +17,10 @@ html {
 
 /** 2 - Headers
 ------------------------------------------------------------*/
-h1,h2,h3,h4,h5,h6 {
+h1, h2, h3, h4, h5, h6 {
   font-family: degular, sans-serif;
   font-weight: 600;
+  margin: 0
 }
 
 h1 {
```

Comment:
This should, ideally, have a `.h1` class too. Alternatively, I think here and below, we can simplify our overall code by not needing so many `@extend`s if we do a `[class^=h1]` instead of `.h1`.